docs/elements/hubspot/tips.md

```diff
@@ -12,7 +12,28 @@ order: 50
 
 ## Tips
```

Comment:
With the recent changes to the styles in the devportal, this should be a heading 1 (just one pound)